internal/install/execution/install_status.go

```diff
@@ -13,18 +14,26 @@ import (
 
 // nolint: maligned
 type InstallStatus struct {
-	Complete          bool                    `json:"complete"`
-	DiscoveryManifest types.DiscoveryManifest `json:"discoveryManifest"`
-	EntityGUIDs       []string                `json:"entityGuids"`
-	Error             StatusError             `json:"error"`
-	LogFilePath       string                  `json:"logFilePath"`
-	Statuses          []*RecipeStatus         `json:"recipes"`
-	Timestamp         int64                   `json:"timestamp"`
-	CLIVersion        string                  `json:"cliVersion"`
-	DocumentID        string
-	targetedInstall   bool
-	statusSubscriber  []StatusSubscriber
-	successLinkConfig types.SuccessLinkConfig
+	Complete            bool                    `json:"complete"`
+	DiscoveryManifest   types.DiscoveryManifest `json:"discoveryManifest"`
+	EntityGUIDs         []string                `json:"entityGuids"`
+	Error               StatusError             `json:"error"`
+	LogFilePath         string                  `json:"logFilePath"`
+	Statuses            []*RecipeStatus         `json:"recipes"`
+	Timestamp           int64                   `json:"timestamp"`
+	CLIVersion          string                  `json:"cliVersion"`
+	HasInstalledRecipes bool                    `json:"hasInstalledRecipes"`
+	HasCanceledRecipes  bool                    `json:"hasCanceledRecipes"`
+	HasSkippedRecipes   bool                    `json:"hasSkippedRecipes"`
+	HasFailedRecipes    bool                    `json:"hasFailedRecipes"`
+	RecipesSkipped      []*RecipeStatus         `json:"recipesSkipped"`
+	RecipesCanceled     []*RecipeStatus         `json:"recipesCanceled"`
+	RecipesFailed       []*RecipeStatus         `json:"recipesFailed"`
+	RecipesInstalled    []*RecipeStatus         `json:"recipesInstalled"`
```

Comment:
The `RecipesSkipped`, `RecipesCanceled`, `RecipesFailed`, and `RecipesInstalled` arrays don't necessarily need to be here, but figured I'd throw the idea out there in case this data structure might be helpful. If not, they can safely be removed. Thoughts?